cmd/src/campaigns_common.go

```diff
@@ -101,18 +101,9 @@ func newCampaignsApplyFlags(flagSet *flag.FlagSet, cacheDir, tempDir string) *ca
 		"If true, errors encountered while executing steps in a repository won't stop the execution of the campaign spec but only cause that repository to be skipped.",
 	)
 
-	// We default to bind workspaces on everything except ARM64 macOS at
-	// present. In the future, we'll likely want to switch the default for ARM64
-	// macOS as well, but this requires access to the hardware for testing.
-	var defaultWorkspace string
-	if runtime.GOOS == "darwin" && runtime.GOARCH == "amd64" {
-		defaultWorkspace = "volume"
-	} else {
-		defaultWorkspace = "bind"
-	}
 	flagSet.StringVar(
-		&caf.workspace, "workspace", defaultWorkspace,
-		`Workspace mode to use ("bind" or "volume")`,
+		&caf.workspace, "workspace", "default",
+		`Workspace mode to use ("default", "bind", or "volume")`,
```

Comment:
Would "auto" be a better name for what's happening?

```suggestion
		&caf.workspace, "workspace", "auto",
		`Workspace mode to use ("auto", "bind", or "volume")`,
```

Reply:
Yeah, I wrestled with that too: the `best` function name was an artefact of a different iteration on the name.

`auto` feels better than `default` to me, so I'm going to commit this. Thanks!